src/eth-signer/test_canister/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "ic_sign_test_canister"
+name = "ic-sign-test-canister"
 version = "0.1.0"
```

Comment:
version and edition should be inherited from workspace

Reply:
Done